### DISCUSSION TITLE
FlipperZero in Action

### DISCUSSION THREAD

**Discussion by kookie77** [djsime1/awesome-flipperzero#101] [Show and tell]
Check out the flipper zero in Action. 

https://youtu.be/m_3twAV1Ca4

**Comment by Danilx8** [comment]
The video seems to be unavailable in some countries... Is there any chance the link will be updated?

**Comment by djsime1** [comment]
The video is completely inaccessible.

**Comment by kookie77** [comment]
Sorry here is a link to view the video from my google drive.

https://drive.google.com/file/d/1dMIv3F4s-PRBFV7cAFGICS5rguUdDQL6/view?usp=drivesdk

**Comment by rsdea** [comment]
Is it worth getting a Flipper Zero? I've seen a lot of cool things you can do with it. Are there other options that can do similar? I think that today's new smart phones should be able to handle everything that a flipper zero can handle as they have much better performance. Only there was a good software? but maybe there is?
I have a Samsung s23 Ultra.
I'm most interested in being able to get into cars and locks in the main socket as I think it's exciting to know that it works.
But does a Flipper Zero really handle everything you see.
Does it handle, for example, a Yaledoorman lock?
​
